<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditorClass</class>
 <widget class="QMainWindow" name="TextEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_all"/>
    <addaction name="actionDate_and_time"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionIncrease_font_size"/>
    <addaction name="actionDecrease_font_size"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionIncrease_font_size"/>
   <addaction name="actionDecrease_font_size"/>
   <addaction name="actionFont"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_all"/>
   <addaction name="actionDate_and_time"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/open.svg</normaloff>:/TextEditor/Resources/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/save.svg</normaloff>:/TextEditor/Resources/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/save.svg</normaloff>:/TextEditor/Resources/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/exit.svg</normaloff>:/TextEditor/Resources/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/undo.svg</normaloff>:/TextEditor/Resources/undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/redo.svg</normaloff>:/TextEditor/Resources/redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/cut.svg</normaloff>:/TextEditor/Resources/cut.svg</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/copy.svg</normaloff>:/TextEditor/Resources/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/paste.svg</normaloff>:/TextEditor/Resources/paste.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/delete.svg</normaloff>:/TextEditor/Resources/delete.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/find.svg</normaloff>:/TextEditor/Resources/find.svg</iconset>
   </property>
   <property name="text">
    <string>Find...</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/replace.svg</normaloff>:/TextEditor/Resources/replace.svg</iconset>
   </property>
   <property name="text">
    <string>Replace...</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/select.svg</normaloff>:/TextEditor/Resources/select.svg</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionDate_and_time">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/date-time.svg</normaloff>:/TextEditor/Resources/date-time.svg</iconset>
   </property>
   <property name="text">
    <string>Date and time</string>
   </property>
  </action>
  <action name="actionIncrease_font_size">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/font-increase.svg</normaloff>:/TextEditor/Resources/font-increase.svg</iconset>
   </property>
   <property name="text">
    <string>Increase font size</string>
   </property>
  </action>
  <action name="actionDecrease_font_size">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/font-decrease.svg</normaloff>:/TextEditor/Resources/font-decrease.svg</iconset>
   </property>
   <property name="text">
    <string>Decrease font size</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/font.svg</normaloff>:/TextEditor/Resources/font.svg</iconset>
   </property>
   <property name="text">
    <string>Font...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/about.svg</normaloff>:/TextEditor/Resources/about.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TextEditor.qrc"/>
 </resources>
 <connections/>
</ui>
